job-server-api/src/main/scala/spark/jobserver/context/SparkContextFactory.scala

```diff
@@ -76,22 +77,23 @@ trait ScalaContextFactory extends SparkContextFactory {
 
   val logger = LoggerFactory.getLogger(getClass)
 
-  def loadAndValidateJob(appName: String,
-                         uploadTime: DateTime,
-                         classPath: String,
-                         jobCache: JobCache): J Or LoadingError = {
+  def loadAndValidateJob(
```

Comment:
I think this is a valid change. The existing indentation is not a multiple of 2 spaces. And this reads better IMO.